controller/engine_image_controller_test.go

```diff
@@ -114,6 +114,17 @@ func createEngineImageDaemonSetPod(name string, containerReadyStatus bool, nodeI
 	return pod
 }
 
+func updateOrAppendCondition(conditions []longhorn.Condition, condition longhorn.Condition) []longhorn.Condition {
```

Comment:
Consider move the function to `types/condition.go`.

Reply:
Sure.